packages/components/popover/src/js/popover.html

```diff
@@ -13,5 +13,5 @@
         ng-attr-id="{{:: $popoverCtrl.id }}"
         ng-include="$popoverCtrl.template">
     </div>
-    <div class="oui-popover__arrow" aria-hidden="true"></div>
+    <div class="oui-popover__arrow" aria-hidden="true" x-arrow></div>
```

Comment:
`x-arrow` ?

Reply:
Previously added in controller, related to Popper.js ?

Reply:
Yes, it says to `Popper.js` what's the arrow of the tooltip/popover, to position it according to its placement and the targeted element (https://popper.js.org/docs/v1/#arrowelement) :)

Reply:
Could you please document this then ?

Reply:
i don't find the information yesterday on the documentation of `Popper.js`... So as @FredericEspiau said, it will be great to document it.